public/stylesheets/itvc.css

```diff
@@ -1018,13 +1018,15 @@ html[data-useragent*='MSIE 10.0'] .flex-container {
     }
     .govuk-table-whatYouOwe td:first-of-type {
         width: 170px;
+        vertical-align: top;
     }
     .govuk-table-whatYouOwe td:nth-of-type(2) {
         width: 550px;
```

Comment:
Should we not add vertical-align to this column as well to maintain consistency?